README.md

```diff
@@ -736,6 +742,154 @@ No errors detected
 
 ```
 
+## `mcci-catena-provision-sigfox.py`
+
+This script communicates with catena to get information for register it in sigfox network via sigfox API, then it loads the catena script to device for complete the provisioning process.
+
+### Required
+
+* Python 3.* (Installation steps [here](https://realpython.com/installing-python/))
+* Install Python packages [pyserial](https://pyserial.readthedocs.io/en/latest/pyserial.html#installation), [requests](https://requests.readthedocs.io/en/master/) and [ruamel.yaml](https://yaml.readthedocs.io/en/latest/) using following commands in terminal/command prompt:
+  1. `pip install pyserial`
+  2. `pip install requests`
+  3. `pip install ruamel.yaml`
+* Sigfox backend user account
+* Catena Script File (It should be placed in the same repository as the script)

```

Comment:
Nobody is going to understand this -- it's out of context if they're just jumping here. Why are we not pointing them at the default, and leaving customization to the advanced section? We should *not* mix application customization with provisioning in the introductory documents. We could say:

> The tool depends on your device being able to get comands over a serial port. When you use the tool, you must give it a script that tells the tool how to communicate with the device. These scripts are traditionally called "Catena script files", and have the extension `.cat`. The tool uses the specified script to Sigfox credentials into the device over the device serial port. The script can also do application-specific setup (so you can combine steps during production); but that's not in scope for this discussion. If you're using MCCI Catena devices, start by using the `catena-sigfox-base.cat` script that accompanies this repository. If you've modified the command language, or are using a different device, you may need to modify the script. In that case, please use `catena-sigfox-base.cat` as a starting point.